velox/exec/PrefixSort.h

```diff
@@ -42,17 +42,17 @@ FOLLY_ALWAYS_INLINE void stdSort(
 }; // namespace detail
 
 struct PrefixSortConfig {
-  PrefixSortConfig(uint32_t maxNormalizedKeySize, uint32_t threshold = 130)
+  PrefixSortConfig(int32_t maxNormalizedKeySize, int32_t threshold = 130)
       : maxNormalizedKeySize(maxNormalizedKeySize), threshold(threshold) {}
 
   /// Max number of bytes can store normalized keys in prefix-sort buffer per
   /// entry.
-  const uint32_t maxNormalizedKeySize;
+  const int32_t maxNormalizedKeySize;
 
   /// PrefixSort will have performance regression when the dateset is too small.
-  /// The threshold is set to 100 according to the benchmark test results by
+  /// The threshold is set to 130 according to the benchmark test results by
```

Comment:
Let's remove this comment and remove hard-coded default value here. Let's have this hard coded and documented in one place, QueryConfig.